superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -80,12 +96,17 @@
 
   .is-cached & {
     background-color: @pink;
-    margin-right: 6px;
   }
 
   .vertical-dots-container & {
     display: block;
   }
+
+  a[role="menuitem"] & {
+    width: 8px;
+    height: 8px;
+    margin-right: 6px;
```

Comment:
what's the `margin-right` for?

Reply:
this is the margin for the big red dot indicating it's cached data. it should be 8px. fixed.